src/components/Svgs/PaginationArrow.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { PseudoBox, IconButton } from '@chakra-ui/core';
+import PropTypes from 'prop-types';
+
+/**
+ * @function PaginationArrow
+ *
+ * @param {function} onClick - Callback function that runs when icon is clicked
+ * @param {boolean} isDisabled - Boolean that determines whether the icon is disabled
+ * @param {string} icon - Name of the svg icon from the theme.js file
+ *
+ */
+
+const PaginationArrow = ({ onClick, isDisabled, icon }) => {
```

Comment:
I would probably change `isDisabled` to `hidden` if you're switching over to hiding the element like the design shows. And instead of `icon` I'd suggest a `direction` prop that just takes `"left"` or `"right"`. With that you can then use the SVGs from the design easily and update the `aria-label` + `title` attributes more easily too (previous and next work as values too).